MaterialMonitor/MaterialMonitor.cs

```diff
@@ -283,7 +283,7 @@ private void incMaterial(string edname, int amount, bool fromLogLoad = false)
 
                 if (ma.maximum.HasValue)
                 {
-                    if (previous <= ma.maximum && ma.amount > ma.maximum)
+                    if (previous <= ma.maximum && ma.amount >= ma.maximum)
```

Comment:
I generally find it clearer to write everything with all the inequalities facing the same way.
Shouldn't this be `previous < ma.maximum && ma.maximum <= ma.amount`?
Some unit tests here would clarify things.

Reply:
Fair enough. Operators sorted and unit test added.